src/InterNations/Grid/Grid.php

```diff
@@ -565,7 +565,9 @@ public function gridResponse(array $parameters = [], string $view = null, Respon
         if (!$view) {
             return $parameters;
         } else {
-            return $this->container->get('templating')->renderResponse($view, $parameters, $response);
+            $response = new Response();

```

Comment:
issue: why are we creating a new response? shouldn't we pass through the response var from method arguments?

Reply:
Yes you are right, that's something that renderResponse checks, and I missed.
I changed it now to make sure we are using the arguments response if it's not null.